### PR TITLE
[4] Non-Controversial CodeStyle Cleanup for /api

### PR DESCRIPTION
**Pull request by PhilETaylor**
[4] Non-Controversial CodeStyle Cleanup for /api

Code review.

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d8eb0976145cf2220b966aed3d24f29cc56caa22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33346">issues.joomla.org/tracker/joomla-cms/33346</a>.</sub>

File: api/components/com_config/src/View/Component/JsonapiView.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function displayList(array $items = null)
 
 		$offset              = $currentPageQuery['offset'];
 		$limit               = $currentPageQuery['limit'];
-		$totalItemsCount     = count($items);
+		$totalItemsCount     = \count($items);
 		$totalPagesAvailable = ceil($totalItemsCount / $limit);
```
Comment (alikon) [inline api/components/com_config/src/View/Component/JsonapiView.php:73]: shouldn't be: 

-  `\ceil()`
-  `\array_splice()`

Comment (PhilETaylor) [reply api/components/com_config/src/View/Component/JsonapiView.php:73]: shouldn't be? They are not. 

Can you explain why they should be? Neither methods are opcode optimised in PHP source code https://github.com/php/php-src/blob/9dc947522186766db4a7e2d603703a2250797577/Zend/zend_compile.c#L4192 and threrefore neither should be root namespaces with a preceding`\` for optimisation.

Comment (alikon) [reply api/components/com_config/src/View/Component/JsonapiView.php:73]: ah ok , thanks

**Comment by PhilETaylor** [comment]
Closing as [no one has the balls](https://github.com/joomla/joomla-cms/pull/33357#issuecomment-827627256) to merge this.

**Pull request closed without merging**